pytermgui/ansi_interface.py

```diff
@@ -69,27 +78,53 @@
     "overline",
 ]
 
+_HEX_REGEX: re.Pattern = re.compile(r"#?[0-9a-zA-Z]{3,6}", re.IGNORECASE)
+"""
+Regex to validate HEX
+"""
+
+_CURSOR_POS_REGEX: re.Pattern = re.compile(r".*\x1b\[(?P<y>\d*);(?P<x>\d*)R")
+"""
+Regex to parse cursor query string
+"""
+
+DECORATION_MODES: dict[str, int] = {
+    "reset": 0,
+    "bold": 1,
+    "dim": 2,
+    "italic": 3,
+    "underline": 4,
+    "blink": 5,
+    "inverse": 7,
+    "invisible": 8,
+    "strikethrough": 9,
+    "overline": 53,
+}
+"""
+Dictionary of decoration modes
+"""
```

Comment:
As I explain above, I appreciate these changes, as well as the documentation added to them. Not happy with the docstring style though, see below.